css/style.css

```diff
@@ -0,0 +1,201 @@
+* {
+  padding:0;
+  margin:0;
+  border: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+@font-face {
+  font-family:'Montserrat';
+  src:url(../fonts/Montserrat-Regular.ttf);
+  font-weight: 400;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 14px;
+}
+
+.container {
+  width:1024px;
+  margin: 0 auto;
+}
+
+header {
+  background: #373c3f;
+  padding: 45px 0 40px 0;
+}
+
+.nav {
+  padding-left: 54px;
+}
+
+.nav li{
+  display: inline-block;
+  list-style: none;
+  margin-right: 30px;
+}
+
+.nav li:last-of-type{
+  margin-right: 0;
+}
+
+.nav li a {
+  font-size: 15px;
+  color: #ebebeb;
+}
+
+.nav li a:hover {
+  color:#fff;
+}
+
+.banner {
+  background: #17b2c5;
+  padding:205px 0 170px 0;
+  color:#fff;
+}
+
+.banner-content {
+  padding: 0 84px;
+}
+
+h1 {
+  font-size: 60px;
+}
+
+.banner a {
+  display: block;
+  color:rgba(255, 255, 255, 0.8);
```

Comment:
Why not rgba? If i need to add white color with transparency 80% like in PDS, what way is better?